includes/admin.inc

```diff
@@ -373,6 +373,13 @@ function islandora_solr_admin_settings($form, &$form_state) {
     '#description' => t('The number of results to show per page.'),
     '#default_value' => variable_get('islandora_solr_num_of_results', '20'),
   );
+  $form['islandora_solr_tabs']['default_display_settings']['islandora_solr_search_navigation'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Enable search navigation block'),
+    '#size' => 5,
```

Comment:
Not used by checkbox https://api.drupal.org/api/drupal/developer!topics!forms_api_reference.html/7#size